src/command_modules/azure-cli-dls/azure/cli/command_modules/dls/_params.py

```diff
@@ -50,8 +50,14 @@
 register_cli_argument('dls fs delete', 'recurse', help='Indicates this should be a recursive delete of the folder.', action='store_true')
 register_cli_argument('dls fs upload', 'overwrite', help='Indicates that, if the destination file or folder exists, it should be overwritten', action='store_true')
 register_cli_argument('dls fs upload', 'thread_count', help='Specify the parallelism of the upload. Default is the number of cores in the local machine.', type=int)
+register_cli_argument('dls fs upload', 'chunk_size', help='Number of bytes for a chunk. Large files are split into chunks. Files smaller than this number will always be transferred in a single thread. Default 268435456', type=int)
+register_cli_argument('dls fs upload', 'buffer_size', help='Number of bytes for internal buffer. This block cannot be bigger than a chunk and cannot be smaller than a block. Default 4194304 ', type=int)
+register_cli_argument('dls fs upload', 'block_size', help='Number of bytes for a block. Within each chunk, we write a smaller block for each API call. This block cannot be bigger than a chunk. Default:4194304 ', type=int)
 register_cli_argument('dls fs download', 'overwrite', help='Indicates that, if the destination file or folder exists, it should be overwritten', action='store_true')
 register_cli_argument('dls fs download', 'thread_count', help='Specify the parallelism of the download. Default is the number of cores in the local machine.', type=int)
+register_cli_argument('dls fs download', 'chunk_size', help='Number of bytes for a chunk. Large files are split into chunks. Files smaller than this number will always be transferred in a single thread. Default 268435456', type=int)
+register_cli_argument('dls fs download', 'buffer_size', help='Number of bytes for internal buffer. This block cannot be bigger than a chunk and cannot be smaller than a block. Default 4194304 ', type=int)
+register_cli_argument('dls fs download', 'block_size', help='Number of bytes for a block. Within each chunk, we write a smaller block for each API call. This block cannot be bigger than a chunk. Default:4194304 ', type=int)
```

Comment:
Can pass in `default` as a keyword argument and just specify the default value.
Then it wouldn't be needed in custom.py either.
This prevents duplicating the value in multiple places.